tests/test_tc201.py

```diff
@@ -49,19 +51,29 @@ def example(x: "something") -> something:
     ),
     (
         textwrap.dedent('''
-    class X:
-        def foo(self) -> 'X':
-            pass
-    '''),
+        class X:
+            def foo(self) -> 'X':
+                pass
+        '''),
         set(),
     ),
     (
         textwrap.dedent('''
-    from __future__ import annotations
-    class X:
-        def foo(self) -> 'X':
-            pass
-    '''),
+        from __future__ import annotations
+        class X:
+            def foo(self) -> 'X':
```

Comment:
I guess this wasn't changed, but I'm a bit stumped as to why this shouldn't produce a TC201

Reply:
I think this is correct, this should only avoid producing a TC101. The TC20x rules don't care about `from __future__ import annotations`.